dts/bindings/arm/nxp,kinetis-mcg.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2019 Vestas Wind Systems A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: NXP Kinetis Multipurpose Clock Generator (MCG)
+version: 0.1
+
+description: >
+    This is a representation of the NXP Kinetis MCG IP node
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "nxp,kinetis-mcg"
+      generation: define
```

Comment:
Please update this to inherit `base.yaml` and remove the now-redundant properties

Reply:
@MaureenHelm Fixed, thanks.